etc/checkfmt.sh

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
 
+go install golang.org/x/tools/cmd/goimports
+
 HAS_FMT_ERR=0
 # For every Go file in the project, excluding vendor...
-for file in $(go list -f '{{$dir := .Dir}}{{range .GoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}' ./...); do
+
+for file in $(go list -f '{{$dir := .Dir}}{{range .GoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}{{range .TestGoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}{{range .IgnoredGoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}{{range .CgoFiles}}{{printf "%s/%s\n" $dir .}}{{end}}' ./... ); do
   # ... if file does not contain standard generated code comment (https://golang.org/s/generatedcode)...
   if ! grep -Exq '^// Code generated .* DO NOT EDIT\.$' $file; then
-    FMT_OUT="$(gofmt -l -d -e $file)" # gofmt exits 0 regardless of whether it's formatted.
```

Comment:
Should we keep the `gofmt` call if `goimports` only checks imports?

Reply:
`goimports` also does formatting . The only thing it is missing is the `-s` argument to `gofmt` - do we want to block PR's on that?

Reply:
See also https://github.com/golang/go/issues/21476

Reply:
Note we weren't blocking on `-s` before.